### PR TITLE
Update renderer according to the post from embed

### LINKED ISSUE #51: When posting from embed with SVG/Canvas, we should use the same renderer

### PR DESCRIPTION
**Pull request by ydlamba**
**Update renderer with the post**

- Fixes #51.

### CODE REVIEW

**Comment by domoritz** [comment]
Thank you. We already have a CYCLE_RENDERER action. Do you think it makes sense to merge them into one?

**Comment by ydlamba** [comment]
Yeah. UPDATE_RENDERER is enough for both required tasks.

File: src/actions/editor.ts [hunk 1]
```diff
@@ -14,10 +14,11 @@ export const SHOW_ERROR_PANE: 'SHOW_ERROR_PANE' = 'SHOW_ERROR_PANE';
 export const SHOW_TOOLTIP: 'SHOW_TOOLTIP' = 'SHOW_TOOLTIP';
 export const TOGGLE_AUTO_PARSE: 'TOGGLE_AUTO_PARSE' = 'TOGGLE_AUTO_PARSE';
 export const UPDATE_EDITOR_STRING: 'UPDATE_EDITOR_STRING' = 'UPDATE_EDITOR_STRING';
+export const UPDATE_RENDERER: 'UPDATE_RENDERER' = 'UPDATE_RENDERER';
 export const UPDATE_VEGA_LITE_SPEC: 'UPDATE_VEGA_LITE_SPEC' = 'UPDATE_VEGA_LITE_SPEC';
 export const UPDATE_VEGA_SPEC: 'UPDATE_VEGA_SPEC' = 'UPDATE_VEGA_SPEC';
 
-export type Action = SetMode | ParseSpec | SetVegaExample | SetVegaLiteExample | UpdateVegaSpec | UpdateVegaLiteSpec | SetGistVegaSpec | SetGistVegaLiteSpec | ToggleAutoParse | CycleRenderer | ShowCompiledVegaSpec | ShowErrorPane | LogError | UpdateEditorString | ShowTooltip | ExportVega;
+export type Action = SetMode | ParseSpec | SetVegaExample | SetVegaLiteExample | UpdateVegaSpec | UpdateVegaLiteSpec | SetGistVegaSpec | SetGistVegaLiteSpec | ToggleAutoParse | CycleRenderer | ShowCompiledVegaSpec | ShowErrorPane | LogError | UpdateEditorString | ShowTooltip | ExportVega | UpdateRenderer;
```
Comment (domoritz) [inline src/actions/editor.ts]: SetRenderer seems more consistent with the other actions.

Comment (ydlamba) [reply src/actions/editor.ts]: Done.

**Pull request merged**